evergreen.yml

```diff
@@ -15,10 +15,28 @@ functions:
       working_dir: src
       script: |
         set -o errexit
-        virtualenv --python=/opt/mongodbtoolchain/v3/bin/python3 venv
+        set -o verbose
+
+        /opt/mongodbtoolchain/v3/bin/python3 -m venv venv
         . venv/bin/activate
 
-        pip install -r requirements.txt
+        pip install poetry
+        poetry install
+
+  deploy:
+    - command: shell.exec
+      params:
+        working_dir: src
+        script: |
+          set -o errexit
+          . venv/bin/activate
```

Comment:
Do we have to activate venv if this is done in the `create virtualenv` step done as part of `pre`?

Reply:
I think so. I believe each command is run in its own shell and `activate` only applies to the current shell.